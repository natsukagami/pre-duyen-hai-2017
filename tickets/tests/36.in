6464 386865700
78 52 94 86 52 8 51 2 7 79 72 45 20 82 48 85 71 34 84 28 43 100 24 76 88 32 8 15 47 29 62 26 73 24 68 66 65 5 10 14 64 2 13 9 32 7 89 74 51 4 29 7 31 37 21 51 22 35 26 45 90 77 39 45 16 49 45 47 89 63 81 77 24 79 29 80 18 11 69 33 11 99 25 12 84 16 90 79 100 74 63 20 79 55 60 46 49 37 86 37 76 62 80 48 66 87 95 22 8 43 40 69 20 2 95 3 33 69 77 33 5 94 38 71 88 77 90 61 4 1 6 50 73 23 87 39 24 94 67 90 94 79 86 37 29 61 30 57 8 24 57 63 42 57 11 24 99 63 71 96 97 15 82 60 24 7 25 41 33 37 66 68 79 21 89 46 9 25 80 77 38 46 81 26 86 35 54 42 7 29 16 56 97 13 37 67 69 14 93 89 48 40 85 39 59 74 1 93 89 53 9 48 2 17 71 29 56 12 23 40 3 81 77 82 30 60 12 8 11 33 5 82 50 24 65 31 32 11 97 53 10 26 52 28 32 41 22 20 55 66 18 20 49 63 88 39 77 62 1 93 84 41 21 92 40 68 14 25 18 26 35 7 34 18 15 31 1 44 96 10 12 4 42 77 44 75 2 46 76 50 55 97 40 28 53 34 16 27 64 97 11 29 86 37 41 49 68 3 62 31 28 75 9 83 35 62 31 67 87 46 39 61 31 46 52 22 22 62 93 82 52 66 33 45 54 78 14 3 17 93 44 9 58 60 29 95 47 38 88 45 89 76 75 41 83 53 86 32 35 2 48 82 17 93 88 39 20 86 91 26 96 36 69 40 98 29 47 37 76 5 14 100 45 71 42 89 62 30 32 18 93 62 29 38 75 27 6 71 40 46 96 59 7 6 3 60 82 52 74 15 67 28 66 4 3 4 83 36 65 85 38 62 89 65 41 56 32 67 57 49 74 27 42 51 50 83 71 57 47 15 66 92 4 93 41 18 57 22 32 14 37 56 41 23 7 35 2 96 82 23 82 12 72 30 14 3 13 23 46 22 58 25 42 35 23 6 82 14 69 14 46 77 39 23 61 22 34 81 65 11 100 79 94 27 35 32 30 33 77 28 26 97 15 96 94 1 56 80 39 20 25 10 24 81 76 41 2 27 34 58 75 100 14 69 28 53 4 16 45 51 68 27 12 80 2 59 2 48 5 63 56 66 16 12 44 61 5 94 40 21 60 60 48 93 36 39 40 42 27 83 4 23 67 77 97 22 81 14 76 31 61 70 36 4 76 22 5 26 99 13 20 40 51 52 94 67 28 50 74 69 32 78 94 57 93 28 23 100 8 5 86 9 34 31 38 97 48 31 82 60 41 21 53 95 10 16 91 1 92 66 71 87 81 59 62 11 20 77 4 62 59 23 85 7 49 16 73 50 64 45 62 11 92 12 41 79 53 62 82 60 46 8 12 54 74 95 26 70 96 93 19 53 21 14 49 55 82 16 56 44 67 8 47 41 57 95 68 36 88 6 82 3 46 85 90 34 88 14 3 59 48 48 53 28 92 89 90 3 76 62 73 17 46 23 64 28 40 26 96 50 57 65 53 12 14 63 87 81 39 29 24 26 16 22 25 18 56 63 29 92 19 22 51 71 25 76 30 72 5 85 97 32 61 82 71 61 77 17 85 19 50 49 75 27 85 98 98 18 84 76 90 28 35 99 94 16 75 84 47 12 45 1 86 2 37 99 32 11 69 87 52 79 65 33 34 3 58 13 39 34 4 20 45 28 42 20 97 44 9 52 80 33 18 88 41 56 69 49 98 86 39 15 89 8 11 5 17 53 61 6 63 11 50 15 23 70 84 90 19 17 76 56 42 16 82 30 61 1 3 30 37 40 14 45 32 84 9 86 100 9 6 83 6 94 59 57 69 64 86 29 65 14 60 8 91 94 2 28 68 95 44 47 25 51 34 52 56 25 55 81 100 48 90 41 17 1 4 74 20 33 29 35 92 72 33 57 8 68 8 59 4 57 95 26 49 48 4 59 44 11 86 5 94 81 13 54 43 21 6 62 96 69 22 76 75 50 99 64 56 9 68 99 66 92 66 66 48 100 72 67 13 9 74 33 62 16 86 1 4 78 48 17 53 29 92 86 81 78 10 15 6 37 12 86 32 87 81 92 22 58 48 15 100 45 55 89 100 12 64 9 65 94 31 3 1 29 16 79 62 31 3 80 18 38 8 48 92 69 31 37 100 56 19 24 44 91 17 21 86 37 30 60 45 31 36 35 48 89 88 42 72 39 23 85 85 39 64 23 11 87 34 25 31 73 46 43 99 68 95 17 35 96 46 67 9 33 17 55 91 87 3 61 4 8 1 49 99 22 10 98 88 46 69 58 67 79 50 24 84 21 74 75 6 93 55 15 10 99 81 10 36 73 92 73 29 9 22 8 33 55 27 21 79 96 44 66 26 22 1 29 16 6 48 6 89 71 95 60 26 75 48 48 31 45 71 52 65 57 93 32 18 27 50 7 62 92 4 86 9 83 32 14 33 20 81 4 59 11 21 65 8 22 54 45 47 81 14 41 69 72 89 99 94 56 42 82 13 76 12 3 66 8 72 69 77 19 55 37 32 75 97 56 81 37 22 71 9 62 18 91 15 20 84 91 83 45 45 85 45 46 72 51 28 76 18 47 97 59 28 92 21 1 36 43 51 73 49 51 75 81 83 62 63 60 92 55 93 3 88 90 95 66 94 38 69 44 94 63 72 62 37 24 26 26 13 15 47 90 9 66 98 70 73 73 65 16 90 68 99 44 24 56 40 18 7 12 15 47 37 79 32 35 26 80 2 65 82 79 52 12 82 80 43 34 71 26 67 11 50 16 13 43 84 50 96 32 88 18 89 93 26 78 76 60 52 25 96 84 39 16 50 3 27 23 80 59 87 19 12 44 42 11 44 27 73 42 17 65 49 22 66 56 50 42 41 42 89 77 27 14 44 34 67 44 61 49 50 85 1 53 95 9 48 3 99 11 77 4 47 22 32 3 22 12 89 28 92 31 58 48 35 17 4 24 90 15 71 86 45 60 73 95 45 91 42 46 18 34 71 81 62 15 26 80 59 51 32 23 83 82 42 79 86 75 92 88 93 65 63 49 14 50 3 29 58 42 71 61 37 28 99 61 21 28 71 3 83 85 96 2 49 32 28 5 4 6 62 4 93 15 91 34 25 49 97 43 85 53 75 48 5 65 57 58 9 36 67 11 10 22 7 50 74 23 57 84 51 88 79 2 78 56 55 40 14 14 4 34 61 57 67 18 86 74 29 56 38 95 72 73 1 60 84 38 38 18 93 44 3 34 95 4 90 51 91 65 70 97 30 38 87 5 70 97 92 86 84 69 23 42 68 58 14 54 81 69 83 2 72 35 49 64 97 66 76 45 31 2 80 36 74 63 29 82 99 34 88 70 16 47 70 69 78 30 31 15 100 84 35 40 44 12 31 51 95 39 27 43 100 40 47 81 79 6 8 37 27 58 13 7 21 71 3 83 46 48 33 79 74 86 64 66 32 41 22 59 35 90 35 99 20 50 50 34 86 63 73 96 22 58 66 86 14 87 55 50 32 61 56 20 61 45 95 63 81 34 92 80 70 74 35 35 8 79 44 10 5 60 67 14 67 94 39 50 40 15 63 24 40 54 58 91 55 23 14 91 7 76 42 63 53 80 65 78 69 68 18 14 12 35 83 36 82 40 53 96 25 80 86 61 23 30 79 16 25 86 1 31 68 29 53 44 42 60 11 54 90 58 75 27 57 77 11 23 39 60 12 82 27 93 52 88 32 68 8 26 78 65 1 85 10 15 19 100 25 83 84 57 83 49 56 81 26 23 58 19 6 75 62 100 47 40 93 82 41 63 75 46 80 36 53 93 60 77 29 58 39 88 77 43 92 18 6 66 1 5 62 27 69 86 100 96 15 60 80 1 54 57 72 94 66 59 8 41 94 53 24 88 94 17 6 45 64 77 20 83 45 47 89 67 45 54 79 94 48 93 64 99 95 59 29 50 35 71 22 8 74 72 34 76 64 67 72 18 19 52 63 13 77 15 96 78 47 88 14 70 40 42 4 72 32 69 36 92 95 50 19 96 61 52 35 25 73 68 12 64 49 44 70 6 51 90 54 5 86 83 85 45 14 74 59 94 63 35 48 39 13 17 98 50 76 76 74 47 65 17 99 36 29 75 94 27 46 28 7 44 30 94 28 27 29 17 57 22 53 97 24 69 40 32 71 71 38 63 77 78 66 42 18 89 47 72 13 44 62 66 67 7 1 3 55 67 4 69 97 13 99 87 23 19 55 4 29 71 12 8 6 96 82 43 76 46 62 24 67 80 68 71 94 49 86 18 39 82 48 40 34 95 56 11 22 16 55 49 7 20 70 26 77 16 6 17 7 75 73 77 80 33 73 74 7 42 30 27 100 41 32 38 50 10 88 94 52 88 3 8 56 99 49 92 2 54 40 39 5 56 82 7 12 77 26 37 29 76 54 49 4 79 18 60 16 100 97 74 3 1 20 49 36 90 31 72 92 61 52 60 74 22 87 31 58 21 88 2 11 34 41 50 70 80 45 2 4 72 56 74 50 78 85 70 69 46 2 95 42 58 95 78 100 18 91 8 24 32 36 47 27 40 78 11 46 14 31 53 61 95 43 72 12 99 3 15 85 88 22 1 73 60 74 30 44 47 37 54 66 66 30 82 7 41 92 4 37 71 94 96 61 80 21 15 98 2 74 86 92 25 39 86 46 33 63 41 70 90 47 95 22 21 53 93 51 49 83 15 97 90 46 86 86 76 18 12 90 46 63 32 19 3 55 13 44 74 66 55 11 57 14 51 30 2 27 47 21 97 80 3 69 50 100 21 24 63 89 79 2 86 82 25 83 51 100 75 13 89 56 91 20 16 14 1 20 9 82 79 52 2 28 77 94 19 82 36 92 7 58 18 44 27 76 62 86 75 34 29 98 61 55 34 75 41 73 99 21 27 27 21 62 74 66 61 76 69 30 21 40 99 29 83 20 34 55 86 64 66 70 24 42 53 73 97 68 19 77 58 8 63 74 37 96 38 38 1 39 26 86 25 52 80 37 61 22 89 14 26 20 49 4 6 92 98 38 81 13 5 58 24 1 12 78 31 23 27 1 95 54 70 5 4 99 58 90 95 56 87 85 40 86 72 49 11 1 47 63 81 11 39 73 64 6 75 53 42 41 75 72 64 95 88 50 40 11 41 80 65 6 56 20 89 40 39 16 6 68 84 54 60 95 52 13 48 90 12 56 17 41 53 74 21 97 88 19 5 83 50 56 61 47 83 24 90 74 52 94 67 91 30 87 51 92 25 8 93 97 42 48 14 52 36 79 70 79 46 78 26 40 17 100 76 44 56 43 27 53 76 72 69 91 61 56 17 56 57 80 32 76 66 60 10 10 66 90 20 83 84 16 22 79 43 77 49 42 25 4 98 43 81 1 15 28 93 18 64 84 25 59 11 36 53 75 35 81 69 33 36 71 27 15 15 36 82 3 14 39 85 77 32 29 79 77 2 76 10 88 33 24 73 82 51 96 89 6 82 6 16 21 13 56 26 77 12 7 79 90 49 20 60 11 50 44 51 88 43 91 93 62 99 9 38 70 41 26 81 99 21 45 17 68 24 90 46 73 69 62 66 57 52 77 63 19 54 7 8 97 44 49 21 21 13 71 65 3 55 92 41 60 60 27 57 66 43 40 91 37 23 75 20 16 31 40 55 60 36 2 45 52 78 10 12 39 12 78 72 86 21 27 28 60 92 81 16 99 1 52 33 84 12 90 98 33 24 3 9 35 9 41 90 18 74 69 86 79 54 95 44 99 44 93 31 93 1 16 93 87 71 37 38 96 21 11 44 78 67 1 9 12 43 6 24 25 18 33 100 99 45 79 70 80 14 81 86 84 86 24 15 2 16 47 88 84 43 75 100 98 58 23 82 64 69 11 5 82 73 79 26 23 78 95 30 25 76 8 32 3 70 66 31 39 69 34 74 13 68 64 99 80 43 83 79 33 97 98 83 66 63 80 8 66 59 69 15 97 94 67 24 6 63 38 87 37 90 85 73 89 79 44 57 91 57 64 62 38 18 60 72 54 30 68 86 20 85 4 83 76 24 86 92 29 80 33 64 93 74 43 21 95 25 61 93 36 24 49 10 72 25 84 12 80 17 87 22 33 88 20 73 29 19 2 47 58 32 46 24 53 8 92 91 65 37 14 50 69 72 78 60 57 34 23 44 43 80 65 46 89 92 37 28 18 77 12 11 94 22 56 87 37 90 14 83 8 13 72 80 82 98 81 67 2 97 87 60 86 64 55 94 77 63 64 100 19 88 58 18 28 73 62 67 6 80 98 92 32 31 62 58 54 6 55 82 66 76 9 3 39 41 53 24 82 52 82 19 56 40 39 29 98 31 98 30 42 57 100 6 11 82 85 84 56 20 84 7 9 64 50 20 51 31 20 26 71 8 25 21 20 78 78 30 39 87 70 59 18 31 39 90 56 98 2 20 17 36 100 17 26 22 58 72 25 2 62 61 20 94 81 48 63 56 33 75 75 4 95 83 20 28 60 28 66 86 70 74 47 82 31 80 72 42 60 33 89 7 5 87 85 60 20 28 98 70 7 38 47 25 50 23 56 15 2 50 65 1 9 24 75 95 77 91 13 79 14 84 54 27 71 24 76 80 55 51 1 40 50 12 75 42 5 91 41 85 75 97 11 59 2 55 59 72 80 29 18 82 56 4 84 84 34 99 90 51 82 40 25 11 42 61 20 34 42 26 32 62 52 2 43 12 83 97 22 71 95 58 96 1 64 99 90 32 1 62 54 1 62 25 60 56 76 5 95 38 2 8 79 14 59 80 12 24 73 39 41 56 13 41 42 74 100 85 40 64 99 2 15 32 37 5 83 30 86 39 100 60 65 7 73 42 89 32 58 35 41 53 48 91 100 20 67 23 24 49 61 96 24 59 91 93 21 84 50 32 38 6 55 10 44 76 79 80 38 26 44 43 69 39 70 84 54 57 82 57 90 94 26 91 78 72 70 33 47 40 38 47 61 44 67 35 20 16 14 90 74 9 49 46 28 92 51 11 85 89 66 93 32 74 36 79 43 30 2 85 21 88 41 96 16 12 61 59 89 9 73 29 87 46 25 10 44 13 72 65 65 20 35 67 94 13 56 41 14 91 34 13 29 69 69 40 12 75 81 88 87 98 56 25 96 100 7 96 4 98 94 99 78 71 33 4 100 27 92 40 67 1 94 44 70 88 37 7 81 12 28 37 45 68 99 100 67 55 66 50 80 5 9 23 73 65 33 38 46 87 63 72 11 50 74 58 41 78 31 66 3 34 98 35 65 94 25 18 9 47 67 53 1 14 83 60 42 74 57 69 17 60 88 63 22 18 13 64 78 31 91 40 46 100 20 34 39 100 88 75 57 90 92 50 9 4 81 24 18 66 74 6 9 8 38 99 100 78 16 68 7 49 62 27 99 42 20 27 90 93 11 82 85 46 79 75 25 72 29 31 92 58 15 36 23 44 23 12 15 74 72 29 8 70 24 2 89 58 51 65 47 29 57 5 38 89 96 56 43 41 55 28 91 94 39 30 5 87 8 43 26 78 69 81 22 86 68 33 94 99 30 90 43 79 36 39 25 86 37 41 12 81 37 71 66 15 92 53 94 58 31 68 36 2 72 12 97 99 32 60 29 64 62 27 60 89 5 86 22 27 60 64 43 26 45 95 29 33 39 69 61 79 50 77 4 68 49 58 81 99 4 10 93 15 2 33 97 88 70 7 47 24 73 19 96 85 41 53 47 20 38 30 7 89 29 28 3 67 60 84 68 76 27 24 28 69 57 2 62 17 57 31 30 22 51 48 63 38 75 89 76 58 33 86 61 22 21 61 61 87 48 98 98 94 60 93 98 47 69 28 72 66 66 75 100 28 87 96 83 39 59 61 98 75 28 10 20 54 52 29 5 94 59 38 81 51 83 89 49 89 50 6 92 72 75 42 65 87 18 74 88 5 26 19 30 59 31 86 17 92 79 59 51 84 22 47 53 10 44 83 88 44 7 79 97 2 45 65 96 62 89 69 23 17 72 94 56 69 24 75 83 74 28 96 32 27 49 12 67 2 35 22 91 12 44 56 61 6 58 75 66 10 85 54 78 81 92 18 59 85 10 67 63 37 13 10 67 44 42 10 81 55 97 82 84 38 83 93 37 84 53 67 74 37 54 42 71 63 94 80 73 36 73 29 75 35 42 87 93 91 69 74 98 95 11 75 9 18 26 56 12 19 51 72 51 45 37 65 17 97 42 31 80 95 87 37 28 21 37 29 45 13 20 32 47 92 9 82 100 36 95 83 97 39 59 40 55 48 63 17 97 50 34 10 18 27 71 92 79 22 78 99 31 69 8 20 10 86 84 53 48 65 55 10 97 73 30 54 37 51 64 39 75 19 87 12 12 46 62 89 27 50 48 100 61 32 6 71 87 3 34 91 52 36 16 74 47 85 50 8 58 38 48 92 11 21 59 77 65 84 14 14 40 28 6 80 66 33 40 16 65 52 78 23 36 78 62 14 8 27 31 100 81 64 41 62 87 27 49 13 59 82 45 50 32 84 64 41 94 10 30 39 61 77 12 58 31 10 47 76 5 68 44 97 13 69 34 53 3 13 32 68 47 20 82 53 17 12 73 55 14 22 19 81 99 53 83 75 80 6 55 89 21 7 43 37 43 99 31 1 25 49 61 29 47 48 80 60 16 1 68 80 44 78 74 63 85 20 20 89 18 78 70 48 67 2 63 21 99 10 46 50 65 14 30 14 48 57 44 44 100 74 35 52 96 42 56 12 42 33 41 45 22 93 62 63 21 85 68 79 85 23 66 93 98 78 77 3 85 23 53 100 100 86 30 18 45 22 74 96 34 40 98 30 88 82 12 33 65 85 78 27 69 93 20 14 72 53 72 100 27 39 65 100 70 5 33 36 17 23 81 84 55 16 35 28 9 47 17 14 22 66 13 2 37 68 62 20 1 47 8 1 19 40 70 1 20 27 40 65 7 8 15 60 37 58 79 67 75 25 72 74 29 32 36 70 27 72 47 49 49 31 98 62 75 91 57 28 52 97 62 6 28 7 96 28 42 66 30 4 91 67 69 62 1 10 87 4 55 38 4 88 53 57 57 77 43 25 59 21 54 88 55 33 66 40 47 9 38 14 2 29 33 68 83 7 18 89 67 93 14 44 58 93 29 88 17 40 16 15 48 59 98 26 28 56 84 13 67 83 46 63 6 81 15 68 47 54 70 83 86 25 34 1 18 54 20 10 65 32 1 7 67 75 94 73 36 48 82 1 88 12 52 59 90 10 69 97 5 24 83 42 42 84 18 13 51 13 67 62 39 2 100 49 15 39 39 45 60 14 5 72 9 98 60 61 48 70 9 6 87 92 97 45 90 30 66 84 11 78 29 66 78 48 65 98 67 44 36 1 81 52 5 33 34 84 47 32 100 68 66 28 31 66 6 9 12 95 28 30 28 51 76 78 58 57 15 97 49 74 21 30 94 96 70 11 2 1 69 68 87 55 89 10 31 2 35 23 71 7 25 71 17 51 98 1 96 8 16 81 76 91 100 64 53 57 92 5 41 59 66 63 22 16 18 45 48 81 19 71 98 91 9 43 73 32 56 79 88 10 62 62 62 100 46 65 12 32 34 98 56 45 92 57 13 49 98 70 25 53 19 63 88 16 54 90 76 44 30 76 76 100 31 62 10 47 36 78 61 18 73 63 9 66 25 88 10 5 41 49 87 55 93 84 93 91 84 82 79 74 80 76 26 14 74 47 56 62 56 66 41 79 24 20 67 96 48 62 78 69 27 83 40 95 44 100 85 37 94 83 56 29 3 57 91 22 65 80 82 39 33 28 14 3 9 26 68 34 6 50 33 52 10 63 73 74 53 24 72 68 11 75 15 75 90 1 71 1 91 14 46 89 7 46 34 35 94 47 4 64 89 31 20 21 11 47 77 20 6 83 42 25 34 38 76 11 10 43 28 47 90 43 10 4 55 70 51 4 4 37 17 61 83 63 40 57 5 20 38 39 3 79 89 75 50 36 34 59 85 7 13 47 67 98 87 47 66 26 40 8 25 53 45 12 71 96 18 28 86 36 45 67 28 84 51 45 6 68 61 67 66 51 33 86 64 84 5 83 28 70 27 19 100 81 35 10 36 91 46 70 100 30 44 48 27 20 94 37 79 9 50 38 37 43 18 6 91 71 97 99 19 48 88 95 13 35 39 59 24 6 61 28 44 25 47 79 36 23 91 33 24 54 12 28 50 64 9 92 87 14 69 95 10 52 99 5 91 86 25 31 60 65 4 84 46 47 27 84 13 100 32 97 85 34 35 61 13 33 89 29 96 63 49 6 34 33 48 100 50 51 41 67 31 10 42 53 90 98 20 37 45 44 88 19 50 23 20 15 27 97 76 5 11 84 63 44 43 57 6 12 44 89 11 50 95 33 61 82 40 33 42 82 1 72 32 68 93 65 66 72 91 14 26 27 29 68 35 27 61 8 57 90 8 10 84 4 89 91 81 56 69 26 72 64 78 25 39 48 31 4 92 33 51 6 22 87 88 59 57 75 45 85 38 77 90 52 20 22 25 68 24 95 14 88 48 80 38 41 94 37 56 21 91 36 71 47 95 40 64 71 46 83 11 90 79 22 28 90 99 85 17 74 69 81 44 24 21 22 40 59 10 58 21 76 66 64 25 61 97 18 29 1 75 35 64 88 50 7 69 66 17 72 79 13 20 42 92 3 76 2 79 17 52 45 54 63 59 63 75 13 38 11 19 19 26 31 100 60 72 99 69 13 45 96 22 89 62 83 86 74 89 2 8 31 94 68 77 92 85 66 88 82 62 27 24 92 51 68 8 82 19 13 43 77 6 32 95 36 70 98 1 70 15 56 69 32 10 76 27 7 37 75 31 7 72 71 17 73 90 79 48 17 57 75 2 78 45 1 24 86 1 58 25 96 96 67 77 84 75 9 95 23 82 49 98 71 19 55 66 52 49 66 1 56 75 21 16 58 67 12 24 92 46 25 74 36 89 81 7 30 25 11 1 71 32 93 26 22 84 91 95 2 32 77 12 61 41 56 83 2 30 91 12 100 18 19 76 16 75 28 62 54 98 92 71 38 83 15 79 66 57 7 3 76 39 28 30 70 3 15 20 49 100 91 95 81 26 49 21 28 53 1 26 29 22 84 12 46 80 22 31 68 41 41 74 47 79 81 59 51 44 3 62 76 6 49 37 3 51 4 54 88 45 99 7 83 58 41 96 95 65 41 38 40 26 11 46 89 12 79 67 74 94 4 57 25 91 54 46 44 82 60 16 6 21 80 70 5 64 83 96 45 28 26 15 7 59 61 85 2 58 31 23 29 73 55 33 71 23 93 26 66 70 95 83 8 58 28 62 88 87 74 86 55 23 29 3 86 87 29 32 72 72 54 28 48 38 19 21 22 3 8 62 5 9 48 63 21 58 78 51 10 78 77 80 55 96 24 34 28 22 33 82 58 52 94 77 47 50 10 56 62 16 24 18 29 28 24 1 12 73 69 98 41 53 88 36 51 68 33 95 42 44 44 29 21 94 15 67 16 79 39 43 85 33 57 54 58 98 62 80 12 30 36 40 44 89 99 27 80 85 17 75 33 83 85 39 94 82 98 85 6 34 15 97 78 1 43 57 81 57 30 90 73 86 96 29 95 58 26 89 17 63 42 75 100 80 66 81 76 22 9 12 62 6 48 43 64 57 13 24 73 70 46 4 22 12 84 71 69 88 5 89 40 64 26 87 23 85 36 84 84 71 95 9 31 93 65 72 23 95 27 6 91 49 81 74 68 81 79 4 90 70 36 19 28 100 48 63 62 81 80 24 41 15 58 6 15 25 17 51 31 50 5 32 18 96 94 36 18 5 89 56 57 74 77 59 100 57 42 11 67 83 69 18 73 32 17 65 59 34 53 35 15 19 53 45 39 1 25 49 32 33 95 23 16 18 81 27 76 58 94 91 21 58 86 66 29 91 20 18 40 61 66 63 85 40 92 54 24 67 30 52 5 46 12 13 37 57 11 46 86 37 28 68 10 23 90 1 37 8 21 17 1 16 99 42 96 78 74 37 54 22 14 87 10 59 31 63 28 44 4 72 66 76 43 37 55 91 46 94 26 43 24 28 69 92 74 52 32 47 41 21 57 24 29 17 17 23 2 97 61 84 55 31 80 51 72 38 68 84 74 6 30 79 89 17 11 56 83 27 85 63 31 5 45 30 36 6 6 49 3 29 47 33 13 21 85 5 84 50 81 51 73 54 32 51 33 12 85 22 11 55 33 70 87 80 76 58 5 61 29 43 50 44 39 71 1 22 76 13 54 84 94 27 36 98 5 39 65 92 98 42 56 62 8 4 39 70 84 48 59 24 72 65 30 25 34 94 46 88 56 68 72 48 90 34 95 83 57 97 93 40 44 45 53 89 49 70 12 48 33 32 56 96 51 20 76 63 56 7 96 54 33 45 81 23 40 25 98 22 75 97 87 11 59 24 46 26 58 90 4 22 89 15 4 48 22 25 58 98 86 43 46 50 67 89 1 20 38 31 26 71 28 64 92 97 18 70 80 8 33 24 94 48 42 77 63 26 62 28 50 3 27 33 57 61 28 25 29 84 80 34 65 71 64 83 12 37 59 92 65 87 37 91 82 24 49 69 90 95 73 68 77 78 81 35 12 95 70 74 93 80 48 14 6 67 45 40 81 12 88 83 28 26 47 58 2 9 58 2 74 6 51 54 4 64 66 94 77 80 27 61 2 27 8 86 8 1 88 23 2 1 87 79 56 28 85 39 64 77 99 67 46 46 15 7 30 51 47 87 88 90 22 8 97 11 26 30 12 93 10 100 25 2 31 56 88 26 89 8 22 65 29 56 4 85 79 100 49 26 17 24 10 62 22 18 65 80 23 19 38 78 55 43 10 18 22 78 65 86 6 33 18 63 34 8 58 48 79 41 79 9 32 96 20 36 46 76 31 71 52 69 38 4 5 21 1 87 11 48 42 23 41 50 77 4 5 71 76 68 69 52 25 39 50 21 86 22 4 90 45 63 80 50 65 32 69 11 84 37 69 3 1 59 63 98 20 53 92 86 81 72 34 10 68 55 99 15 12 85 41 36 29 26 59 76 98 87 71 69 34 88 90 3 66 21 97 93 47 92 21 19 5 46 52 59 67 33 19 6 41 59 8 81 38 7 29 74 53 4 40 19 85 65 79 44 83 88 11 84 9 47 32 26 70 82 76 5 91 21 11 87 3 65 44 30 54 1 58 69 39 63 27 41 33 35 64 86 22 78 13 75 71 83 32 24 15 90 100 91 14 48 29 68 9 4 83 93 49 74 66 96 67 2 97 23 23 95 25 32 57 14 71 19 67 25 45 89 90 12 98 99 87 38 10 97 25 41 65 64 7 74 51 88 67 34 53 88 23 82 18 45 81 36 68 96 96 50 33 92 69 21 3 93 15 5 45 35 80 6 2 84 96 29 56 75 49 29 27 3 80 70 69 42 28 41 61 63 39 22 55 52 87 7 100 8 39 22 1 77 60 97 36 77 22 37 98 7 65 39 26 19 10 42 65 14 22 4 98 29 80 27 1 39 26 43 53 59 2 47 19 75 21 27 79 24 52 9 55 78 76 7 19 53 16 62 61 60 75 70 43 90 22 72 73 36 11 93 27 19 55 23 59 69 76 77 23 16 33 14 32 28 35 85 11 99 1 17 96 54 77 19 41 79 57 12 25 69 100 22 4 100 46 19 34 87 91 73 89 69 41 88 77 15 54 13 94 96 98 62 78 23 95 89 80 71 80 70 39 37 21 55 24 67 82 41 23 75 50 67 84 91 7 48 23 30 47 97 79 14 67 49 44 56 12 60 6 99 52 42 15 61 69 60 40 84 90 71 30 10 36 88 19 14 47 9 14 10 95 94 53 91 87 32 70 60 76 94 80 77 92 43 3 29 21 28 81 46 78 50 80 81 92 36 91 99 3 98 11 10 11 5 47 62 88 46 82 75 83 13 84 98 40 7 42 41 35 91 2 29 65 56 17 29 40 3 62 60 28 96 16 90 59 13 12 36 4 2 87 25 61 79 64 55 92 54 30 29 81 52 34 83 18 46 64 98 22 55 54 62 89 98 59 63 4 75 81 53 61 26 1 57 5 44 35 100 6 73 4 44 53 80 46 8 69 56 28 52 55 96 63 95 40 58 57 64 31 36 94 30 61 33 55 7 1 62 86 1 41 80 64 10 14 7 59 19 30 32 37 22 22 97 88 7 21 44 43 63 14 17 35 88 37 76 52 76 54 34 61 93 30 66 74 90 27 65 26 42 7 33 12 15 41 74 95 71 50 8 32 15 72 56 42 5 87 56 19 76 10 31 23 17 24 97 10 52 34 74 65 18 86 100 83 15 45 
