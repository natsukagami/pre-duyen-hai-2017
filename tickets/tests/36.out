213957594706660
